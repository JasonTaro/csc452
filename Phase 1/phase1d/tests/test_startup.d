test_startup.o: tests/test_startup.c ../phase1.h \
  /Users/kyle/include/usloss.h /Users/kyle/include/usyscall.h
