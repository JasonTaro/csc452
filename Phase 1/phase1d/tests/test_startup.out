P2_Startup
Sentinel quitting.
No runnable processes, halting.
